### DISCUSSION TITLE
How can I query simulation results？

### DISCUSSION THREAD

**Discussion by hrluo93** [LohseLab/gIMble#120] [Q&A] [answered]
Dear glMble!

Thanks a lot development such a good tool!

I used below commands

gimble blocks -z gmanalysis.z -l 64
gimble windows -z gmanalysis.z -w 500 -s 100
gimble info -z gmanalysis.z
gimble tally -z gmanalysis.z -k 2,2,2,2 -l blocks_kmax2 -t blocks
gimble tally -z gmanalysis.z -k 2,2,2,2 -l windows_kmax2 -t windows
gimble optimize -z gmanalysis.z -l IM_AB_optimize -d tally/windows_kmax2 -w -m IM_AB -r A -u 2.58e-09 -A 10_000,2_500_000 -B 10_000,1_000_000 -C 1_000_000,5_000_000 -M 0,1e-5 -T 0,5_000_000 -g CRS2 -e 19 -i 10_000
gimble makegrid -z gmanalysis.z -m IM_AB -b 64 -r A -u 2.58e-09 -k 2,2,2,2 -A=10000,2435834,12,lin -B=10000,1000000,12,lin -C 1000000,5000000,12,lin -T 5000000 -M 0,1E-05,16,lin -e 19 -l IM_AB_grid
gimble gridsearch -z gmanalysis.z -g makegrid/IM_AB_grid -d tally/windows_kmax2 -p 18

gimble simulate -z gmanalysis.z --seed 19 --replicates 1 --windows 152609 --blocks 500 --block_length 64 -a 10 -b 12 --gridsearch_key gridsearch/windows_kmax2/IM_AB_grid -k 2,2,2,2 -s IM_AB_grid -p 40 -u 2.58e-9
**However**, when I want to get me window bed result, I performed: 
gimble query -z gmanalysis.z -l simulate/IM_AB_grid
_It showed an error_: [X] 'simulate/IM_AB_grid' does not point to a tally.

Did I missing some steps?

Best wishes!

**Comment by DRL** [comment, marked as answer]
Hi @hrluo93,

The problem for tallies is that you have one per replicate. So if you want for example the tally of the `0`-th replicate you need to specify the label as:

```
gimble query -z gmanalysis.z -l simulate/IM_AB_grid/0
```

Sorry for not making this clearer. Let me know if this works as expected.

cheers, 

dom

**Comment by hrluo93** [comment]
> 

Hi, thanks a lot !
I have got the grid tsv, but how can I get the window Me bed like in the example. It seems that the usage guildlines end in simulate step？

My grid tsv like this:

window_idx count het_b het_a het_ab fixed-diff   It seems that not the genome coords and Me? 

Best wishes！

**Comment by DRL** [comment]
I think you are looking for the gridsearch results. Just query them with the label that you got after doing the `gridsearch` step ...
